src/crewai/cli/test_violations.py

```
@@ -0,0 +1,57 @@
+
+class DataProcessor():                             # Empty parentheses not needed
+  def __init__(self):
+    self.data=[]                              # Missing space around operator
+    self.processed=False                       # Missing space around operator
+
+  def add_data(self,item:Any)->None:            # Missing space after comma, inconsistent type hint spacing
+    self.data.append(item)
+
+  def process(self)->List[Dict]:                # Incomplete type hint
+    for i in range(len(self.data)):           # Non-Pythonic loop
+      item=self.data[i]
```

Ruff

Local variable `item` is assigned to but never used

Remove assignment to unused variable `item`

(F841)